cdc/log.cc

```diff
@@ -989,10 +992,12 @@ cdc::cdc_service::impl::augment_mutation_call(lowres_clock::time_point timeout,
                 // Note: further improvement here would be to coalesce the pre-image selects into one
                 // iff a batch contains several modifications to the same table. Otoh, batch is rare(?)
                 // so this is premature.
+                tracing::trace(tr_state, "CDC: Selecting preimage [mutation #{}]", idx);
                 f = trans.pre_image_select(qs.get_client_state(), db::consistency_level::LOCAL_QUORUM, m);
             }
 
-            return f.then([trans = std::move(trans), &mutations, idx] (lw_shared_ptr<cql3::untyped_result_set> rs) mutable {
+            return f.then([trans = std::move(trans), &mutations, idx, tr_state = std::move(tr_state)] (lw_shared_ptr<cql3::untyped_result_set> rs) mutable {
+                tracing::trace(tr_state, "CDC: Generating log mutation [mutation #{}]", idx);
```

Comment:
How are these indexes of any use?
Maybe we should show the pk, or token, or even both.